webui/react/src/pages/JobQueue/JobQueue.tsx

```diff
@@ -117,8 +118,9 @@ const JobQueue: React.FC = () => {
         }
       },
     };
-    if (isJobOrderAvailable &&
-        job.summary.jobsAhead > 0 && canManageJob(job, selectedRp)) {
+    if (selectedRp && isJobOrderAvailable &&
+        job.summary.jobsAhead > 0 && canManageJob(job, selectedRp) &&
+        !unsupportedQPosition.has(selectedRp.schedulerType)) {
       triggers[JobAction.MoveToTop] = () => moveJobToPosition(jobs, job.jobId, 1);
```

Comment:
**question:** I just noticed that `moveJobToPosition` is defined as an async function, should this trigger also be an async function just like the `Kill` function defined right below?

Reply:
I'll go ahead and land as is so it can get included in the release. I'll update one direction or the other if we go that route.